datajunction-server/datajunction_server/sql/decompose.py

```diff
@@ -104,6 +104,8 @@ def _simple_associative_agg(self, func) -> list[MetricComponent]:
             )
 
         expression = str(arg)
+        # TODO: we should only hash on the lowercase version of the expression, but we can't
+        # do this migration until after we get versioned measures
```

Comment:
PR is ready to fix that once this is in: https://github.com/DataJunction/dj/pull/1451